perfzero/lib/cloud_manager.py

```diff
@@ -133,9 +134,37 @@ def create(username, project, zone, machine_type, accelerator_count,
   logging.info('Successfully created gcloud computing instance %s with %s accelerator.\n',  # pylint: disable=line-too-long
                instance_name, accelerator_count)
 
+  # Wait until we can ssh to the newly created computing instance
+  cmd = 'gcloud compute ssh {} --project={} --zone={} --strict-host-key-checking=no --command="ls"'.format(instance_name, project, zone)  # pylint: disable=line-too-long
+  ssh_remaining_retries = 12
+  ssh_error = None
+  while ssh_remaining_retries > 0:
+    ssh_remaining_retries -= 1
+    try:
+      run_command(cmd, is_from_user=False)
+      ssh_error = None
+    except Exception as error:  # pylint: disable=W0703
+      ssh_error = error
+      if ssh_remaining_retries:
+        logging.info('Cannot ssh to the computing instance. Try again after 5 seconds')  # pylint: disable=line-too-long
+        time.sleep(5)
+      else:
+        logging.error('Cannot ssh to the computing instance after 60 seconds due to error:\n%s', str(ssh_error))  # pylint: disable=line-too-long
```

Comment:
I think `logging.info` by default adds a newline at the end of message. Consider adding the newline in your custom formatter instead of manually type `\n` after every message?

Reply:
Yeah logging.info() will add a new line at the end of the message. So if we do `logging.info('message1')` followed by `logging.info('message2')`, the output will be 

```
message1
message2
```

I am not sure I fully understand the comment here. We are not manually typing `\n` after every message here, right? Could you clarify what is the recommended change to the custom formatter here?